torch/_dynamo/variables/builder.py

```diff
@@ -1002,6 +1004,9 @@ def wrap_to_fake_tensor_and_record(
                 source=source,
             )
         )
+        for k, v in e.__dict__.items():
+            if k not in fake_e.__dict__:
+                fake_e.__dict__[k] = v
```

Comment:
This is a weird thing to do and it has the potential to corrupt fake tensor state if the user tensor redefines an attribute fake tensor uses internally.

Reply:
hmm, I could make a weak mapping between real<>fake, I suppose, we just need a way to launder it through